wire-library/wire-tests/src/commonTest/proto-kotlin/com/squareup/wire/protos3/kotlin/all_types/AllTypes.kt

```diff
@@ -0,0 +1,1390 @@
+// Code generated by Wire protocol buffer compiler, do not edit.
```

Comment:
this package is protos3. Should it be protos3 or proto3? I think we use both and should not!

Reply:
fixed